src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -276,13 +281,13 @@ private IEnumerable<IssueMarker> RemoveSuppressedIssues(IEnumerable<IssueMarker>
 
         private void RefreshIssues()
         {
-            UpdateIssues(this.Factory.CurrentSnapshot.IssueMarkers ?? Enumerable.Empty<IssueMarker>());
+            UpdateIssues(Factory.CurrentSnapshot.IssueMarkers ?? Enumerable.Empty<IssueMarker>());
         }
 
         private void UpdateIssues(IEnumerable<IssueMarker> issueMarkers)
         {
-            var oldSnapshot = this.Factory.CurrentSnapshot;
-            var newSnapshot = new IssuesSnapshot(this.ProjectItem.ContainingProject.Name, this.FilePath, oldSnapshot.VersionNumber + 1, issueMarkers);
+            var oldSnapshot = Factory.CurrentSnapshot;
+            var newSnapshot = new IssuesSnapshot(ProjectName, FilePath, oldSnapshot.VersionNumber + 1, issueMarkers);
```

Comment:
The `ProjectItem` instance variable should now be redundant and can be removed.

Reply:
Already did